src/libexpr/eval.cc

```diff
@@ -1621,15 +1621,15 @@ void EvalState::callFunction(Value & fun, size_t nrArgs, Value * * args, Value &
                 return;
             } else {
                 /* We have all the arguments, so call the primop. */
-                auto name = vCur.primOp->name;
+                auto fn = vCur.primOp;
```

Comment:
IIUC:

```suggestion
                auto * fn = vCur.primOp;
```

Relevant when reviewing for performance.